lib/shortcode/jetpack-portfolio.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+$numberOfColumns = array();
+for ( $i = 1; $i <= 6; $i++ ) {
+	$numberOfColumns[ $i ] = $i;
+}
+
+$displayOrder['author'] = __( 'Author', 'pbsandwich' );
+$displayOrder['date'] = __( 'Post Date', 'pbsandwich' );
+$displayOrder['title'] = __( 'Post Title', 'pbsandwich' );
+$displayOrder['rand'] = __( 'Randomized', 'pbsandwich' );
+
+$displayDir['ASC'] = __( 'Ascending', 'pbsandwich' );
+$displayDir['DESC'] = __( 'Descending', 'pbsandwich' );
```

Comment:
Put all code above inside the function. `$numberOfColumns` and `$displayOrder` should not be global variables